x-pack/test/apm_api_integration/trial/tests/service_maps/service_maps.ts

```diff
@@ -287,34 +288,20 @@ export default function serviceMapsApiTests({ getService }: FtrProviderContext)
     });
 
     describe('when there is data with anomalies', () => {
-      before(() => esArchiver.load('ml_8.0.0'));
-      after(() => esArchiver.unload('ml_8.0.0'));
+      before(() => esArchiver.load('apm_8.0.0'));
+      after(() => esArchiver.unload('apm_8.0.0'));
 
       it('returns service map elements', async () => {
-        const response = await supertest.get(
-          '/api/apm/service-map?start=2020-08-26T11%3A00%3A00.000Z&end=2020-08-26T11%3A30%3A00.000Z'
-        );
+        const start = encodeURIComponent('2020-09-10T06:00:00.000Z');
+        const end = encodeURIComponent('2020-09-10T07:00:00.000Z');
+
+        const response = await supertest.get(`/api/apm/service-map?start=${start}&end=${end}`);
 
         expect(response.status).to.be(200);
-        const opbeansJavaWithAnomaly = response.body.elements.filter(
-          (el: { data: { id: string } }) => el.data.id === 'opbeans-java'
+        const anomalies = response.body.elements.filter(
+          (el: { data: { serviceAnomalyStats?: {} } }) => !isEmpty(el.data.serviceAnomalyStats)
```

Comment:
can you add a check for values actually being populated (ie, non-null/undefined)?